apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/RestTemplateFactory.java

```diff
@@ -59,8 +60,14 @@ public boolean isSingleton() {
   }
 
   public void afterPropertiesSet() throws UnsupportedEncodingException {
+
+    PoolingHttpClientConnectionManager connectionManager = new PoolingHttpClientConnectionManager();
+    connectionManager.setMaxTotal(portalConfig.connectPoolMaxTotal()); //最大连接数
+    connectionManager.setDefaultMaxPerRoute(portalConfig.connectPoolMaxPerRoute()); //每个路由（域名）最大连接数
```

Comment:
Please provide your comments in English.

Reply:
Maybe I can remove this comment, I think the variable name is clear enough